Runtimes/Supplemental/Runtime/CMakeLists.txt

```diff
@@ -93,7 +93,8 @@ add_compile_options(
   "$<$<COMPILE_LANGUAGE:Swift>:-cxx-interoperability-mode=default>"
   "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-Xfrontend -experimental-spi-only-imports>"
   "$<$<AND:$<BOOL:${${PROJECT_NAME}_ENABLE_LIBRARY_EVOLUTION}>,$<COMPILE_LANGUAGE:Swift>>:-enable-library-evolution>"
-  "$<$<AND:$<BOOL:${${PROJECT_NAME}_ENABLE_PRESPECIALIZATION}>,$<COMPILE_LANGUAGE:Swift>>:SHELL:-Xfrontend -prespecialize-generic-metadata>")
+  "$<$<AND:$<BOOL:${${PROJECT_NAME}_ENABLE_PRESPECIALIZATION}>,$<COMPILE_LANGUAGE:Swift>>:SHELL:-Xfrontend -prespecialize-generic-metadata>"
+  "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-Xfrontend -define-availability -Xfrontend \"Backtracing 6.2: macOS 26.0\">")
```

Comment:
This should not be added here. We auto-generate the availability macros in https://github.com/al45tair/swift/blob/c74154533b3e648a8f8e31f4e655e77da0042eb0/Runtimes/Supplemental/cmake/modules/AvailabilityMacros.cmake

Please add this to utils/availability-macros.def instead

Reply:
Not this one we don't, by design.

Reply:
See the code in the other `CMakeLists.txt` for why.